app/inbound/calculation/save-placeholder-organisation.js

```diff
@@ -0,0 +1,11 @@
+const db = require('../../data')
+
+const savePlaceholderOrganisation = async (organisation, sbi, transaction) => {
+  await db.organisation.findOrCreate({
```

Comment:
As you're doing this by PK you _could_ use `upsert` rather than `findOrCreate`.  However, I don't actually know if there's any actual benefit to doing that so more just for info.